dpsim-models/src/EMT/EMT_Ph1_Switch.cpp

```diff
@@ -0,0 +1,135 @@
+/* Copyright 2017-2021 Institute for Automation of Complex Power Systems,
+ *                     EONERC, RWTH Aachen University
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *********************************************************************************/
+
+#include <dpsim-models/EMT/EMT_Ph1_Switch.h>
+
+using namespace CPS;
+
+// !!! TODO: 	Adaptions to use in EMT_Ph1 models phase-to-ground peak variables
+// !!! 			with initialization from phase-to-phase RMS variables
+
+///FIXME: Inconsistent naming in ph1 and p3 switches: (mIsClosed, mSwitchClosed)
+
```

Comment:
TODO comment was copied over from "EMT::Ph3::Switch", FIXME was added by me since the variable has different names in e.g. EMT::Ph3::Switch and DP::Ph1::Switch